main.js

```diff
@@ -6,3 +6,64 @@ JS:
 4. bonus: clicking on the translucent background closes the dialog
 5. bonus: use Class
 */
+function dialogBox($component) {
+  const questionContainer = $component.find('.question');
+  const questionContainerHtml = questionContainer.html();
+  const socialItem = $('.social-link');
+  const buttonOk = $component.find('.button--ok');
+  const buttonCancel = $component.find('.button--cancel');
+  const dialogContainer = $('.dialog');
+  let socialLink = socialItem.attr('data-url');
```

Comment:
There's no point putting this in this scope. This will be overwritten after each click, so could be a `const` inside event-handling function.